src/java_bytecode/java_object_factory.cpp

```diff
@@ -74,7 +71,7 @@ class java_object_factoryt
     const exprt &expr,
     const typet &target_type,
     bool create_dynamic_objects,
-    update_in_placet);
+    update_in_placet update_in_place);
```

Comment:
Only reason would be greater consistency - can undo if you think it is a backward step?